apps/vaccines-updates-client/src/index.html

```diff
@@ -8,6 +8,9 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="icon" type="image/x-icon" href="favicon.ico" />
+  <script crossorigin="anonymous"
```

Comment:
Why like this and not to use the FBLogin service created in the project?

Reply:
Diverge that created because Gil you worked in parallel on FB login.
I assume that soon we'll need non FB login as well and we'll need a refactoring here, so let's defer the merge to this refactoring.